reference/v8.json

```diff
@@ -158,17 +158,17 @@
     "maxzoom": {
       "type": "number",
       "default": 14,
-      "doc": "Maximum zoom to preserve detail at."
+      "doc": "Maximum zoom level at which to create vector tiles (higher means greater detail at high zoom levels but slower performance)."
```

Comment:
Actually I'm not even sure it affects performance in a noticeable way. We should probably bump the default to 16 or even 18.